src/main/java/budgetbuddy/model/transaction/TransactionList.java

```diff
@@ -13,6 +13,7 @@
  */
 public class TransactionList implements Iterable<Transaction> {
 
+    public static final String MESSAGE_CONSTRAINTS = "TransactionList can not be null";
```

Comment:
I'm not too sure why this is here, since the list should technically be nullable.